wrappers/python/CMakeLists.txt

```diff
@@ -17,8 +26,11 @@ if (NOT hasParent)
     option (BUILD_WRITERS "Build with writer support (encoders)" ON)
     option (BUILD_READERS "Build with reader support (decoders)" ON)
     set(BUILD_DEPENDENCIES "AUTO")
-    if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/core)
```

Comment:
Looking at this line, it occurred to me that is does not make any sense in current HEAD because `core` always exists. Then I realized: your function is not necessary at all. Simply do the `IS_SYMLINK` check on `core` (which will fail in an sdist build) and if it is a symlink, then hardcode the `../../core` path. Less lines of code to worry about.

Reply:
Will change, but maybe not today.

Reply:
Updated, I still kept the EXISTS check but made a variable `CORE_PATH` that is set conditionally based on whether `./core`  is a symlink.